products/views.py

```diff
@@ -0,0 +1,72 @@
+import random
+
+from django.views import View
+from django.http import JsonResponse
+
+from . import models
+from reviews import models as reviews_models
+
+
+class MainView(View):
+    def get(self, request):
+        data = models.Product.objects.filter(is_main=True).values(
```

Comment:
`Product`클래스를 바로 import 하셨다면, 클래스를 바로 사용하실 수 있습니다.
```suggestion
        data = Product.objects.filter(is_main=True).values(
```